packages/components/src/mobile/inserter-button/style.native.scss

```diff
@@ -33,11 +33,6 @@
 	height: 32px;
 	justify-content: center;
 	align-items: center;
-	fill: $gray-dark;
-}
-
-.modalIconDark {
-	fill: $white;
```

Comment:
We no longer require to define the fill color here because now that we're using the `BlockIcon` component for rendering the icon, these colors are already used as the default color:
https://github.com/WordPress/gutenberg/blob/55bf89b1c84aa5a1ddd17dc92909fdae4cc11a73/packages/block-editor/src/components/block-icon/style.native.scss#L1-L7